tests/devices/test_default_mixed_torch.py

```diff
@@ -718,3 +718,31 @@ def circuit(weights):
 
     assert isinstance(weights.grad, torch.Tensor)
     assert weights.grad.shape == weights.shape
+
+
+class TestMeasurements:
+    """Tests for measurements with default.mixed"""
+
+    @pytest.mark.parametrize(
+        "measurement",
+        [
+            qml.counts(qml.PauliZ(0)),
+            qml.counts(wires=[0]),
+            qml.sample(qml.PauliX(0)),
+            qml.sample(wires=[1]),
```

Comment:
Nice! Might be worth updating the title to mention that `sample` is being supported too :rocket: